stubs/Handler/CurlMultiHandler.php

```diff
@@ -9,7 +9,7 @@
 class CurlMultiHandler
 {
     /**
-     * @param array{handle_factory?:CurlFactoryInterface,select_timeout?:int|float} $options
+     * @param array{handle_factory?:CurlFactoryInterface,select_timeout?:int|float,options?:array} $options
```

Comment:
```suggestion
     * @param array{handle_factory?:CurlFactoryInterface, select_timeout?:int|float, options?:array} $options
```

Could you also format all the `array{}` types to include a space after a colon? Like this one, to maintain code code style for types, ha-ha.